### PR TITLE
Adds a link to the assessment page that whatsdue scrapes from

### LINKED ISSUE #457: Add the link for the assessment calendar that is scraped in !whatsdue
As part of the !whatsdue command we scrape a url to determine when assessment is due. It might be nice to also return this to the user so they can bookmark it or similar

### PR DESCRIPTION
**Pull request by Guthers**
Closes #457

### CODE REVIEW

File: uqcsbot/utils/uq_course_utils.py [hunk 1]
```diff
@@ -165,13 +166,24 @@ def is_assessment_after_cutoff(assessment, cutoff):
...
+def get_course_assessment_page(course_names: List[str]) -> str:
+    """
+    Determines the course ids from the course names and returns the
+    url to the assement table for the provided courses
+    """
+    profile_ids = map(get_course_profile_id, course_names)
+    return BASE_ASSESSMENT_URL + ','.join(profile_ids)
+
+
+def get_course_assessment(course_names, cutoff=None, assement_url=None):
```
Comment (nicklambourne) [inline uqcsbot/utils/uq_course_utils.py]: typo in assessment

File: uqcsbot/scripts/whatsdue.py [hunk 2]
```diff
@@ -66,4 +68,5 @@ def handle_whatsdue(command: Command):
     if not is_full_output:
         message += ('\n_Note: This may not be the full assessment list. Use -f'
                     + '/--full to print out the full list._')
+    message += f'\nLink to assesment page <{asses_page}|here>'
```
Comment (nicklambourne) [inline uqcsbot/scripts/whatsdue.py]: typo in assessment

**Comment by Guthers** [comment]
@nicklambourne as requested
image

**Comment by Guthers** [comment]
Potentially more useful screenshot
image

**Pull request merged**